src/artemis/experiment_plans/optimise_attenuation_plan.py

```diff
@@ -58,35 +86,240 @@ def arm_devices(xspress3mini, zebra):
     yield from arm_zebra(zebra)
 
 
+def get_new_direction(direction: Direction, deadtime, deadtime_threshold):
+    if direction == Direction.POSITIVE:
+        if deadtime >= deadtime_threshold:
+            direction = Direction.NEGATIVE
+    return direction
+
+
+def calculate_new_direction(direction: Direction, deadtime, deadtime_threshold):
+    if direction == Direction.POSITIVE:
+        if deadtime > deadtime_threshold:
+            direction = Direction.NEGATIVE
+    return direction
```

Comment:
Should: These functions are basically the same, only one is used, can we remove the other?

Reply:
Yes, looks like i accidentally (basically) duplicated that function